packages/elements/src/components/table/table.stories.mdx

```diff
@@ -395,6 +726,7 @@ type Row = {
   expandableContent: React.ReactNode
 }
 ```
+##### Inside cells array
 
 ```
 type Cell = {
```

Comment:
If we have an ArgsTable, we can get rid of the inline types in the MD

Reply:
I couldn't get the argsTable component to work with interfaces or types. If it does work with those then happy days!